Lib/test/test_dataclasses.py

```diff
@@ -2822,17 +2827,9 @@ def test_classvar(self):
                         'typing. ClassVar[str]',
                         'typing.ClassVar [str]',
                         'typing.ClassVar [ str]',
-
-                        # Not syntactically valid, but these will
-                        #  be treated as ClassVars.
-                        'typing.ClassVar.[int]',
-                        'typing.ClassVar+',
                         ):
             with self.subTest(typestr=typestr):
-                @dataclass
-                class C:
-                    x: typestr
-
+                C = dataclass(type("C", (), {"__annotations__": {"x": typestr}}))
```

Comment:
Why this change? Doesn't the code that counteracts double stringification take care of this? (Ditto a few times below.)

Reply:
We can no longer put the value we want since the annotation will always be a static `typestr`